src/components/layout/FooterSection.vue

```diff
@@ -50,13 +50,33 @@
         <img src="../../assets/iconVenture.svg">
       </a>
     </div>
+    <div
+      v-if="!isLoggedIn"
+      class="ww-login-footer"
+    >
+      <a
+        :href="createDeepLinkUrl({ type: 'address' })"
+        class="button"
+        :title="$t('components.layout.RightSection.LoginWithWallet')"
+      >
+        {{ $t('components.layout.RightSection.LoginWithWallet') }}
```

Comment:
`RightSection` is not related to this component. This is why I think lang keys should be structured by terms semantic (if it is necessary) instead of the component hierarchy.

Reply:
The footer is visualized when the mobile navigation is opened so we could login with web-wallet on mobile. This button is also visualized in the RightSection. I could duplicate the strings for this particular case, thus it would follow the semantic strings approach.